spec/dummy_controller_spec.rb

```diff
@@ -45,6 +45,10 @@
       it 'responds with next_page' do
         expect(response_body(response)['next_page']).to be 2
       end
+
+      it 'responds with previous_page' do
```

Comment:
You repeated all this tests... Put in a shared example for this and include it

Reply:
I repeated it because also all of the other tests (that check presence of the parameters in the response) were repeated in almost all of the specs. But you're totally right, so I will use a shared_example not just for my own `previous_page`, but for every test that is being repeated.

Reply:
I think we could use a shared example that groups current/previous/next page logic and pass the page number by parameter.
Someting like:
``` ruby
shared_examples 'respond with correct page numbers' do |previous, current, next|
# some specs
end

#and then call like
include_examples 'respond with correct page numbers', 1,2, 3 
```

Reply:
Yep, I agree with that, good idea